.github/workflows/c-cpp_tmpl.yaml

```diff
@@ -28,17 +31,27 @@ jobs:
         env:
           cmake_version: ${{ matrix.cmake_version }}
           gcc_version: ${{ matrix.gcc_version }}
-        uses: NWChemEx-Project/.github/actions/get_dependencies@master
+          clang_version: ${{ matrix.clang_version }}
+        uses: NWChemEx-Project/.github/actions/get_dependencies@parallel-template
```

Comment:
shouldn't this be master?

Reply:
Sorry, I forgot to delete parallel-build-test.yaml file. No, extra file will be required at this point. All change is in c-cpp_tmpl.yaml. @ryanmrichard

Reply:
And yeah those should be master. @ryanmrichard

Reply:
@ryanmrichard for the ${{inputs.compiler}} ${{inputs.compiler_version}} variables maybe I will have to check a bit more. Do you want to have those as part of this Pull Request? Because these need to be then dynamically set as environment variables. I am not sure to what extent github workflows will allow that.

Reply:
You can do it later. What you have now is fine, it sets the API up.